packages/victory-brush-container/src/brush-helpers.ts

```diff
@@ -150,7 +150,7 @@ const Helpers = {
       const fullDomainBox =
         targetProps.fullDomainBox || this.getDomainBox(targetProps, fullDomain);
       const constrainedBox = this.constrainBox(pannedBox, fullDomainBox);
-      return Selection.getBounds({ ...constrainedBox, scale, horizontal });
+      return Selection.getBounds({ ...constrainedBox, scale });
```

Comment:
We should keep `horizontal` in here ... the `getBounds` method should be updated instead.

Reply:
Ahh, I didn't see `horizontal` was being used by `getDataCoordinates` initially. Updated!